lib/shelley/test/data/cardano-node-shelley/shelley-genesis.yaml

```diff
@@ -31,7 +31,7 @@ protocolParams:
   maxBlockHeaderSize: 217569
   keyDeposit: 1000000
   keyDecayRate: 0
-  nOpt: 3
+  nOpt: 2
```

Comment:
I think it's pretty hard to use the local cluster and integration tests for testing this change.

Testing the new pool listing on existing mainnet pools and comparing with the old implementation will be more fruitful.